src/test/java/com/notably/logic/parser/NotablyParserTest.java

```diff
@@ -85,79 +66,103 @@ public void parseCommand_existingNoteInput_throwsError() throws Exception {
 
     @Test
     public void parseCommand_newCommandInputJump_false() throws Exception {
-        List<? extends Command> commands = parser.parseCommand("new -t CS2103 -b Hi");
+        List<? extends Command> commands = parser.parseCommand("new -t CS2103");
 
         assertTrue(commands.size() == 1);
     }
 
     @Test
     public void parseCommand_newCommandInputJump_true() throws Exception {
-        List<? extends Command> commands = parser.parseCommand("new -t CS2103 -b Hi -o");
+        List<? extends Command> commands = parser.parseCommand("new -t CS2103 -o");
         assertTrue(commands.size() == 2);
     }
```

Comment:
I think it'd be better if you can test out executing these commands and see what they do!

Reply:
done!